avaframe/ana3AIMEC/aimecTools.py

```diff
@@ -121,6 +121,8 @@ def fetchReferenceSimNo(avaDir, inputsDF, comModule, cfgSetup):
         colorVariation: boolean
             True if a color variation should be applied in the plots
     """
+
+    cfgSetup = cfg['AIMECSETUP']
```

Comment:
is it a good idea to give the full cfg instead of the cfgSetup? an option could be to pass the cfgSetup in input and an optional cfg['filter]

Reply:
why is this better?, I thought adding a second input parameter to the function is not needed if we just pass the full cfg